modules/prometheus-server-metrics.adoc

```diff
@@ -0,0 +1,19 @@
+//module included in the following assemblies:
+// * configuration/monitor-acs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="prometheus-server-metrics_{context}"]
+= Monitoring RHACS by using a Prometheus server
```

Comment:
The key difference is that the new metrics are product metrics, and not RHACS infrastructure. Not sure about the wording.

```suggestion
= Monitoring RHACS product metrics by using a Prometheus server
```